src/icons/general/IconChatRound.js

```diff
@@ -2,28 +2,25 @@
 import * as React from 'react';
 
 import AccessibleSVG from '../accessible-svg';
+import type { Icon } from '../flowTypes';
 
-type Props = {
-    className?: string,
-    color?: string,
-    height?: number,
-    /** A text-only string describing the icon if it's not purely decorative for accessibility */
-    title?: string | React.Element<any>,
-    width?: number,
-};
-
-const IconChatRound = ({ className = '', color = '#999', height = 24, title, width = 24 }: Props) => (
+const IconChatRound = ({ className = '', color = '#999', height = 24, title, width = 24 }: Icon) => (
     <AccessibleSVG
         className={`icon-chat-round ${className}`}
         height={height}
         title={title}
-        viewBox="0 0 24 24"
+        viewBox="0 0 32 32"
```

Comment:
It looks like this icon is used in the Sidebar nav as the Activity icon. Does it still look the same with these changes? Could you post a quick screenshot comparison?

Reply:
Yep, definitely...

Here's the theme overridden version: 
<img width="408" alt="screen shot 2019-03-05 at 2 02 54 pm" src="https://user-images.githubusercontent.com/1280912/53840621-73d7ce80-3f4f-11e9-83ce-413b1ba98b7a.png">

And regular:
<img width="404" alt="screen shot 2019-03-05 at 2 05 44 pm" src="https://user-images.githubusercontent.com/1280912/53840774-d630cf00-3f4f-11e9-8904-4e8b76f4010f.png">

Reply:
To clarify, these are both screenshots from **after** my change...